documentation/asciidoc/computers/compute-module/cm-emmc-flashing.adoc

```diff
@@ -4,6 +4,8 @@ The Compute Module has an on-board eMMC device connected to the primary SD card
 
 Please also read the section in the xref:compute-module.adoc#datasheets-and-schematics[Compute Module Datasheets]
 
+IMPORTANT: For mass provisioning of CM3, CM3+, CM4 and Raspberry Pi 4B the https://github.com/raspberrypi/cmprovision[Raspberry Pi Compute Module Provisioning System] is recommended.
```

Comment:
I don't think this actually mass-provisions the Raspberry Pi 4B; only the Compute Modules?

Reply:
The docs claim it does

Reply:
The way **I** read the docs, is that it only _runs_ on a Raspberry Pi 4B, it doesn't actually _provision_ a Raspberry Pi 4B? I guess @ghollingworth can probably clarify.

Reply:
No, I think you're right. Although, I imagine that the provisioning system doesn't actually care what it's provisioning if you had somehow attached EMMC or NVME to a Pi4B. Will update